start.sh

```diff
@@ -1,28 +1,31 @@
 #!/bin/bash
 
-PRINTMODE=""
-while getopts ":hp:" OPTION
-do
-    case $OPTION in
-        h)
-            /launch_df -h
-            exit 1
-            ;;
-        p)
-            PRINTMODE=$OPTARG
-            ;;
-    esac
-done
 
 /usr/bin/supervisord
+set -x
 
-# Enable auto backups using gitwatch of the save directory is a git repo
```

Comment:
I left this out because I like the one-life concept of rogue-likes and this works against that. Could be an optional env variable if there were a desire for it.